surround/runners.py

```diff
@@ -0,0 +1,23 @@
+from abc import ABC, abstractmethod
+
+
+class Runner(ABC):
```

Comment:
I can't find the Runner class implemented anywhere. Is this interface a WIP?

Reply:
There is an example showing how this is used